docs/beginner-checks.md

```diff
@@ -0,0 +1,77 @@
+
+If vulnerabilities are found in your dependencies, there are a few options:
+
+- Update the dependency to a non-vulnerable version (if available)
+- Submit a patch to the vulnerable project
+- Replace the dependency with a non-vulnerable dependency
+- Remove the dependency and write code to take its place
+- If you are sure a vulnerability does not impact your project, you may ignore the dependency by creating an [osv-scanner.toml](https://google.github.io/osv-scanner/configuration/#ignore-vulnerabilities-by-id) file in the same directory as your dependency manifest
+
+If you have handled the vulnerabilities in your dependencies and are still not satisfied with your score for this check, make sure there are no open, unfixed vulnerabilities in your project’s own codebase. Once you have dealt with those, your score should improve.
+
+Next, Scorecard’s [Dependency-Update-Tool check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool) encourages developers to keep their dependencies up to date, which is a great way to stay on top of security updates. This check awards a high score to a project if it uses a dependency update tool such as [Dependabot](https://docs.github.com/en/code-security/dependabot), [Renovate bot](https://docs.renovatebot.com/), or [PyUp](https://github.com/pyupio/pyup#readme). Using one of these tools helps streamline security processes by notifying you when vulnerabilities have surfaced in your dependencies or when new versions of your dependencies become available.
+
+Automated processes like these save you time and are highly configurable; for example, you can set your bot to update dependencies every day or every week at the same time.
+
+If you want to increase your score in this category, sign up for automatic updates with a dependency update tool. Do note, however, that this check can only assess whether the dependency update tool is enabled, not that it is running. To maximize your benefit from this check, be sure that you consistently run your dependency update tool.
+
+### Token-Permissions check helps you secure your workflows
+
+We suggest addressing the [Token-Permissions check](https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions) next because it takes just a few minutes to “set it and forget it” and secure your workflows. The check warns you when your project’s tokens have `write` access instead of `read` access. Projects that use tokens with `write` access for their automated workflows may be vulnerable to malicious code execution. By simply switching `write` to `read` you’ll be able to close off a powerful attack vector (exploiting compromised tokens that have write permissions and using them to inject malicious code into the project).
+
+To change the default setting for token permissions, add the following to the top of your workflow:
+
+```
+permissions:
+contents: read
```

Comment:
this is missing an indentation for `contents`

Reply:
Thanks, added